src/main/java/software/amazon/awssdk/crt/http/Http2Stream.java

```diff
@@ -7,24 +7,37 @@
 
 import software.amazon.awssdk.crt.CrtRuntimeException;
 
+import java.util.concurrent.CompletableFuture;
+
 public class Http2Stream extends HttpStream {
 
     protected Http2Stream(long ptr) {
         super(ptr);
     }
 
     /**
-     * Reset the HTTP/2 stream.
-     * Note that if the stream closes before this async call is fully processed, the RST_STREAM frame will not be sent.
+     * Reset the HTTP/2 stream. Note that if the stream closes before this async
+     * call is fully processed, the RST_STREAM frame will not be sent.
      *
-     * @param http2_stream HTTP/2 stream.
-     * @param http2_error aws_http2_error_code. Reason to reset the stream.
+     * @param errorCode aws_http2_error_code. Reason to reset the stream.
      */
     public void resetStream(final Http2ClientConnection.Http2ErrorCode errorCode) {
-        throw new CrtRuntimeException("Unimplemented");
+        http2StreamResetStream(getNativeHandle(), errorCode.getValue());
     }
 
     /**
-     * @TODO getters for reset stream. Not sure anyone needs it.
+     * TODO: getters for reset stream. Not sure anyone needs it though.
      */
+    private static native void http2StreamResetStream(long http_stream, int errorCode);
+
+    @Override
+    public void writeChunk(final byte[] chunkData, boolean isFinalChunk,
+            final HttpStreamWriteChunkCompletionCallback chunkCompletionCallback) {
+        throw new RuntimeException("Not implemented");
+    }
+
+    @Override
+    public CompletableFuture<Void> writeChunk(final byte[] chunkData, boolean isFinalChunk) {
+        throw new RuntimeException("Not implemented");
+    }
```

Comment:
HTTP/2 may have a different API, and they're not called chunks

maybe solve this by having StreamBase, and only the HTTP/1 version has writeChunk() <-- this is your idea